src/main/java/datawave/query/util/OrdinalDayOfYear.java

```diff
@@ -0,0 +1,53 @@
+package datawave.query.util;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.GregorianCalendar;
+
+public class OrdinalDayOfYear {
+    private String mmDD;
+    private int ordinalDay;
+    private int year;
+    private boolean isLeapYear = false;
+    private GregorianCalendar gregorianCalendar;
+    
+    public static final int[] MONTH_LENGTH = new int[] {31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};
```

Comment:
Are these constants still needed?

Reply:
I'll take a look and remove possible construction debris.

Reply:
I moved the statics to the test class where they get used and removed construction debris.